src/blocks/content-helper/components/related-top-post-list-item.tsx

```diff
@@ -2,35 +2,35 @@
  * External dependencies
  */
 import { __ } from '@wordpress/i18n';
-import { Button, Card, CardBody, CardHeader } from '@wordpress/components';
 
 /**
  * Internal dependencies
  */
 import { RelatedTopPostData } from '../models/related-top-post-data';
+import ViewsIcon from '../icons/views-icon';
+import PublishedLinkIcon from '../icons/published-link-icon';
 
 interface RelatedTopPostListItemProps {
 	post: RelatedTopPostData;
 }
 
-function RelatedTopPostListItem( { post }: RelatedTopPostListItemProps ) {
+function RelatedTopPostListItem( { post }: RelatedTopPostListItemProps ): JSX.Element {
 	return (
-		<>
-			<Card size="small" elevation={ 2 } className="parsely-content-helper-card">
-				<CardHeader><b>{ post.title }</b></CardHeader>
-				<CardBody>
-					<ul>
-						<li>{ __( 'Views:', 'wp-parsely' ) } { post.views }</li>
-						<li>{ __( 'Published:', 'wp-parsely' ) } { post.date }</li>
-						<li>{ __( 'Author:', 'wp-parsely' ) } { post.author }</li>
-					</ul>
-					<p>
-						<Button href={ post.url } target="_blank" variant="primary">{ __( 'Open Post', 'wp-parsely' ) }</Button> { ' ' }
-						<Button href={ post.statsUrl } target="_blank" variant="secondary">{ __( 'Post Stats', 'wp-parsely' ) }</Button>
-					</p>
-				</CardBody>
-			</Card>
-		</>
+		<div className="parsely-top-post" data-testid="parsely-top-post">
```

Comment:
The component is called a list item, but it's not marked up as a list item? This should be `li`, and with the `ol` wrapping it in the RelatedTopPostList component since it's an ordered list.